sphinxgallery/gen_rst.py

```diff
@@ -373,7 +387,7 @@ def execute_script(code_block, example_globals, image_path, fig_count,
     cwd = os.getcwd()
     # Redirect output to stdout and
     orig_stdout = sys.stdout
-
+    figure_list = []
```

Comment:
Nice catch, thanks.

I guess that means that examples without plots are currently broken ...